mysql-compatibility.md

```diff
@@ -27,7 +27,6 @@ aliases: ['/docs-cn/dev/mysql-compatibility/','/docs-cn/dev/reference/mysql-comp
 * 触发器
 * 事件
 * 自定义函数
-* 外键约束 [#18209](https://github.com/pingcap/tidb/issues/18209)
```

Comment:
L178-181 need to be removed or the description needs to be modified.

```
- `foreign_key_checks`：
    + TiDB 默认：`OFF`，且仅支持设置该值为 `OFF`。
    + MySQL 5.7 默认：`ON`。
```

Reply:
great, done.